docs/reference/repo_properties.rst

```diff
@@ -0,0 +1,162 @@
+
+Package Repository Properties
+*****************************
+
+.. _ppa-properties:
+PPA properties
+==============
+
+The following properties are supported for PPA-type repositories:
+
+- type
+   - Type: enum[string]
+   - Description: Specifies type of package-repository, must currently be
+     ``apt``
+   - Examples: ``type: apt``
+- ppa
+   - Type: string
+   - Description: PPA shortcut string
+   - Format: <ppa-owner>/<ppa-name>
+   - Examples:
+      - ``ppa: snappy-devs/snapcraft-daily``
+      - ``ppa: mozillateam/firefox-next``
+
+
+.. _deb-properties:
+Deb properties
+==============
+
+The following properties are supported for Deb-type repositories:
+
+- architectures
+   - Type: list[string]
+   - Description: Architectures to enable, or restrict to, for this repository
+   - Default: If unspecified, architectures is assumed to match the host’s
+     architecture
```

Comment:
This is only the primary architecture, right? If I have multiarch enabled it's not going to add the foreign architectures?

Reply:
Right! Added this clarification, thanks